tools/stores_l10n/add_new_product.md

```diff
@@ -0,0 +1,261 @@
+
+Pay attention to the log when testing a new template that you’ve copied from another product, some functions and variables are exclusive of a specific store, for example:
+* `$keywords_warning` is defined only for App Stores projects.
+* `$short_desc` is defined only for Play Store projects.
+
+## Add views
+
+The next step is to add a view for each channel supported for this product. Views are stored in `views/product_name/channel`, in case of Focus for Android `views/focus_android/release`. There are 2 files to create:
+* `locale_view.php`: used for *General view*.
+* `locale_view_escaped.php`: used for *Description raw HTML*. It’s the same content as `locale_view.php`, but each element is included in `<pre contenteditable="true"></pre>` elements.
+
+As for templates, the easiest way to start is by copying files from an existing product.
+
+This is how the first part of `locale_view.php` looks like:
+
+```PHP
+<h1>Focus for Android Listing Copy (<?= $request['locale'] ?>)</h1>
+
+<h3>Title &mdash; <?= $title_warning ?></h3>
+<pre><?= $app_title($translations) ?></pre>
+
+<h3>Short Description &mdash; <?= $short_desc_warning ?></h3>
+<pre><?= $short_desc($translations) ?></pre>
+```
+
+As you can see, it simply calls the variables created inside the template. The interesting part is that you can add more complex logic in this file:
+
+```PHP
+<?php
+    /*
+        Check if the file used for screenshots exists, display this section
+        only in that case.
+    */
+    $screenshot_lang = $project->getLangFiles($request['locale'], $request['product'], $request['channel'], 'screenshots');
+    if ($screenshot_lang) {
+        $locale_file = LOCALES_PATH . $request['locale'] . '/' . array_shift($screenshot_lang);
+        if (file_exists($locale_file)) {
+            ?>
+            <h3>Screenshots</h3>
+            <pre class="text-center"><?= $screenshots($translations) ?></pre>
+<?php
+
+        }
+    }
+?>
+```
+
+In this case, the screenshots session is displayed only if the requested locale actually has screenshots, i.e. if the file specified in the configuration is available.
+
+`locale_view_escaped.php` is exactly the same as the default view, only using a different HTML structure to make copy and paste easier. For example, the same section displayed above becomes:
+
+```PHP
+<h1>Focus for Android Listing Copy (<?= $request['locale'] ?>)</h1>
+
+<h3>Title &mdash; <?= $title_warning ?></h3>
+<pre contenteditable="true"><?= htmlspecialchars($app_title($translations)) ?></pre>
+
+<h3>Short Description &mdash; <?= $short_desc_warning ?></h3>
+<pre contenteditable="true"><?= $short_desc($translations) ?></pre>
+```
+
+As for the templates, pay attention to differences between a product shipping on Play Store and one shipping on App Stores:
+* For Android products there should be length warnings on title, short description, and description.
+* There are no keywords in Android, short description is not available for iOS products.
```

Comment:
indeed, I wasn't planning to be exhaustive, but maybe it's not a bad to idea to add all cases.